data/tools/incident_response.json

```diff
@@ -137,6 +137,15 @@
       "language": "Go",
       "price": "Free",
       "online": "False"
-    }
+    },
+    {
+      "name": "Snort",
```

Comment:
> Items should be sorted alphabetically

https://inventory.raw.pm/docs/docs/contribution_guidelines/format.html

Reply:
I make the suggested changes could please review it @noraj